Build error on termux

**Describe the bug**
Build fails (used to work!)

**To Reproduce**
Steps to reproduce the behavior:
try to build 86box git on Termux
(or may be unix in general?)

**Expected behavior**
Build finishes

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Termux aarch64
 - 86Box version: b750471e5c886cd90ee69b6f68e8941f4606c9e3

**Additional context**
Build fails with:

```
[ 93%] Built target vid
/data/data/com.termux/files/home/86Box/src/unix/unix.c:634:18: warning: cast to 'void *' from smaller integer type 'int' [-Wint-to-void-pointer-cast]
        header = (void *) (flags & MBX_ANSI) ? "86Box" : L"86Box";
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/data/data/com.termux/files/home/86Box/src/unix/unix.c:634:46: warning: pointer type mismatch ('char *' and 'unsigned int *') [-Wpointer-type-mismatch]
        header = (void *) (flags & MBX_ANSI) ? "86Box" : L"86Box";
                                             ^ ~~~~~~~   ~~~~~~~~
/data/data/com.termux/files/home/86Box/src/unix/unix.c:636:43: warning: cast to smaller integer type 'int' from 'void *' [-Wvoid-pointer-to-int-cast]
        header = (void *) plat_get_string((int) header);
                                          ^~~~~~~~~~~~
/data/data/com.termux/files/home/86Box/src/unix/unix.c:638:44: warning: cast to smaller integer type 'int' from 'void *' [-Wvoid-pointer-to-int-cast]
        message = (void *) plat_get_string((int) message);
                                           ^~~~~~~~~~~~~
/data/data/com.termux/files/home/86Box/src/unix/unix.c:974:25: error: use of undeclared identifier 'EMU_NAME'
                        EMU_NAME, EMU_VERSION_FULL, EMU_GIT_HASH, ARCH_STR, DYNAREC_STR);
                        ^
/data/data/com.termux/files/home/86Box/src/unix/unix.c:974:35: error: use of undeclared identifier 'EMU_VERSION_FULL'
                        EMU_NAME, EMU_VERSION_FULL, EMU_GIT_HASH, ARCH_STR, DYNAREC_STR);
                                  ^
4 warnings and 2 errors generated.
make[2]: *** [src/unix/CMakeFiles/plat.dir/build.make:76: src/unix/CMakeFiles/plat.dir/unix.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1174: src/unix/CMakeFiles/plat.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

--- a/src/unix/unix.c
+++ b/src/unix/unix.c
@@ -36,6 +36,7 @@
 #include <86box/unix_sdl.h>
 #include <86box/timer.h>
 #include <86box/nvr.h>
+#include <86box/version.h>
 #include <86box/video.h>
 #include <86box/ui.h>
 #include <86box/gdbstub.h>

